Client/Constants.h

```diff
@@ -10,16 +10,37 @@ inline constexpr auto MAC_ADDR_STR_SIZE = 17;
 
 inline constexpr auto SERVICE_UUID = "96CC203E-5068-46ad-B32D-E316F5E069BA";
 
+inline unsigned char SERVICE_UUID_IN_BYTES[] = { // this is the SERVICE_UUID but in bytes
+	0x96, 0xcc, 0x20, 0x3e, 0x50, 0x68, 0x46, 0xad,
+	0xb3, 0x2d, 0xe3, 0x16, 0xf5, 0xe0, 0x69, 0xba
+};
+
 inline constexpr auto APP_NAME_W = L"Sony Headphones App";
 
 using Buffer = std::vector<char>;
 
 enum class DATA_TYPE : char
 {
+	DATA = 0,
 	ACK = 1,
-	DATA_MDR = 12
+    DATA_MC_NO1 = 2,
+    DATA_ICD = 9,
+    DATA_EV = 10,
+	DATA_MDR = 12,
+    DATA_COMMON = 13,
+    DATA_MDR_NO2 = 14,
+    SHOT =  16,
+    SHOT_MC_NO1 =  18,
+    SHOT_ICD =  25,
+    SHOT_EV =  26,
+    SHOT_MDR =  28,
+    SHOT_COMMON =  29,
+    SHOT_MDR_NO2 = 30,
+    LARGE_DATA_COMMON =  45,
+    UNKNOWN = -1
```

Comment:
well, I experimented a bit with the decompiled apk and added it in case we want to add features.
I can remove it if you want though it does no harm, does it?

Reply:
I guess, no problem